src/components/main/footer/Footer.jsx

```diff
@@ -13,29 +14,47 @@ const footerTextStyle = {
 
 const Footer = ({
   lastUpdated,
-}) => (
-  <footer
-    className="navbar has-navbar-fixed-bottom"
-    style={{
-      position: 'fixed',
-      bottom: '0',
-      height: '45px',
-      background: '#002449',
-      // Really high z-index here to ensure Footer is on top of modal
-      zIndex: '20000',
-    }}
-  >
-    <div className="level has-text-centered">
-      <div className="level-item">
-        <p style={footerTextStyle}>
-          Data Updated Through:
-          &nbsp;
-          {lastUpdated && moment(1000 * lastUpdated).format('MMMM Do YYYY, h:mm:ss a')}
-        </p>
-      </div>
-    </div>
-  </footer>
-);
+}) => {
+  const location = useLocation();
+
+  const renderFooter = () => {
```

Comment:
Just want to confirm my understanding of this. Will this let us add other switch case(s) where `location.pathname === /about` or `/contact` that sets toRender to the /about or /contact specific footer content and renders it on those routes?